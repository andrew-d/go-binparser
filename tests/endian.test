[
    {"name": "UintTest", "endian": "big",
     "fields": [
        {"name": "default", "offset": 0, "type": "uint32"},
        {"name": "override_big", "offset": 4, "type": "uint32", "endian": "big"},
        {"name": "override_little", "offset": 8, "type": "uint32", "endian": "little"}
    ]}
]
