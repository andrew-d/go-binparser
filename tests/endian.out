Default,%!s(uint32=305419896)
Override_Big,%!s(uint32=305419896)
Override_Little,%!s(uint32=2018915346)
